src/providers/fail_over.c

```diff
@@ -1657,6 +1748,17 @@ fo_get_server_port(struct fo_server *server)
     return server->port;
 }
 
+int
+fo_get_server_family(struct fo_server *server)
+{
+    if (server != NULL &&
+        server->common != NULL &&
+        server->common->rhostent) {
```

Comment:
This is cosmetic and I'm not asking you to fix it. But if you want to, I will not oppose.

You compare against `NULL` in the two previous lines, but not in this one.